### PR TITLE
Create Article.md

### PR DESCRIPTION
**Pull request by TheREveNGers10**
My Active Directory article

### CODE REVIEW

**Review by AviusX — CHANGES_REQUESTED** [review]
Just 3 minor changes-
1. Adding spaces after each full stop in the "What is Active Directory" paragraph.
2. The last line of the "Domain Controller" paragraph: "... the entire network is toast **and** you could do anything you want." (adding the and)
3. The last line of the "Trees" paragraph: "...domains fall into a parent-child relationship, **where** the child domain **takes** on the name of the parent." (Changing those 2 words for better grammar)

**Comment by AviusX** [comment]
@Saket-Upadhyay Tagging for peer review purposes.

File: Submissions/2020-09-13_Active-Directory-Basics/Article.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+    
+# Active Directory Basics
+
+### What is Active Directory(AD)
+**Active Directory** is a directory service developed my Microsoft to manage  Windows domain networks. It stores information related to objects, such as Computers, Users, printers etc. Think of it as a phone book for windows.This enables a user to have one username and password and authenticate over the network using those credentials.Authentication is done using **Kerberos tickets** on Windows devices.**Active Directory** is most commonly used identity management service in the world and can be exploited without the need of exploits instead we abuse features, trusts, policies and more.
```
Comment (AviusX) [inline Submissions/2020-09-13_Active-Directory-Basics/Article.md:5]: Adding spaces here.

**Comment by AviusX** [comment]
Looks good to me.

**Comment by Saket-Upadhyay** [comment]
@TheREveNGers10  please follow directory structure [HERE](https://frigidsec.github.io/blog/2020/08/30/dinosaurs.html) under title **GitHub Structure**

Missing files :
> Status.txt
> rnotes.txt

if you want to add any custom image header include that image with the proper source citation in 
> image directory

@AviusX after this you can mark your review changes in `rnotes.txt`

**Pull request merged**